code/radar/radarsetup.cpp

```diff
@@ -484,6 +484,7 @@ void HudGaugeRadar::drawRange()
 		break;
 
 	default:
+		mprintf(("HUD_config.rp_dist=%d\n", HUD_config.rp_dist));
 		Int3();	// can't happen (get Alan if it does)
```

Comment:
This should be turned into an Error, but this is perhaps the wrong place to do it. This value is only set to something unsafe when it is being read from a savefile, any error checking should take place there.